packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1120,6 +1121,36 @@ async function handler(
     users = [...availableUsers.filter((user) => user.isFixed), ...luckyUsers];
   }
 
+  const attachNonUserBookUid = async (bookerEmail: string, dateFrom: Dayjs, dateTo: Dayjs) => {
```

Comment:
Why is that needed? Shouldn't `reqBody.bookingUid` be already set

Reply:
attachNonUserBookUid function is responsible for attaching bookingUid if non user booker is trying to book on the same time slot.